### PR TITLE
Add react tables

### PR DESCRIPTION
**Pull request by clairesunstudio**
table-mono

### CODE REVIEW

File: react/src/components/datavis/DataTable/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactTable from 'react-table';
+
+import '../../../../node_modules/react-table/react-table.css';
```
Comment (smurrayatwork) [inline react/src/components/datavis/DataTable/index.js]: This shouldn't use a relative path. You should be able to do 'react-table/react-table.css';

File: react/src/components/datavis/DataTable/county.data.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
+[
```
Comment (avrilpearl) [inline react/src/components/datavis/DataTable/county.data.json]: can we rename this to knobs options to match the other components

File: react/src/components/datavis/DataTable/styles.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    }
+  }
+
+  $alignment: left center right;
+  @each $option in $alignment {
+    .data-align-#{$option} {
+      text-align: #{$option};
+    }
+  }
+}
```
Comment (avrilpearl) [inline react/src/components/datavis/DataTable/styles.scss]: can you rename the town.data.json to knobs options to match other components.

File: react/src/components/datavis/DataTable/DataTable.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+### Description
```
Comment (avrilpearl) [inline react/src/components/datavis/DataTable/DataTable.md]: can we rename this folder in the components from `datavis` to `dataviz` to match storybook syntax.

**Review by avrilpearl — CHANGES_REQUESTED** [review]

File: react/src/components/datavis/DataTable/styles.scss [hunk 5]
```diff
@@ -0,0 +1,17 @@
+.ReactTable {
```
Comment (avrilpearl) [inline react/src/components/datavis/DataTable/styles.scss]: can we reduce the font-size in the table? i.e. set it to 14px;

**Review by avrilpearl — APPROVED** [review]
looks good! just respond to steven's comment

**Review by smurrayatwork — APPROVED** [review]

**Pull request merged**